### PR TITLE
Update wallet to actix 0.11

### PR DESCRIPTION
**Pull request by tmpolaczyk**

### CODE REVIEW

File: futures_utils/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,276 @@
+#![allow(clippy::type_complexity)]
+use actix::fut::IntoActorFuture;
+use actix::{Actor, ActorFuture};
+use pin_project_lite::pin_project;
+use std::future::Future;
+use std::pin::Pin;
+use std::task;
+use std::task::Poll;
```
Comment (lrubiorod) [inline futures_utils/src/lib.rs:8]: fmt :P

File: net/Cargo.toml [hunk 2]
```diff
@@ -6,14 +6,14 @@ edition = "2018"
 workspace = ".."
 
 [dependencies]
-actix = { version = "0.8.3", default-features = false }
+actix = { version = "0.11.0-beta.1", default-features = false }
```
Comment (lrubiorod) [inline net/Cargo.toml:9]: is safu use beta?

**Review by lrubiorod — APPROVED** [review]

File: net/Cargo.toml (hunk 2 above)
Comment (tmpolaczyk) [reply net/Cargo.toml:9]: We could merge this PR into a different branch and keep testing it until the final actix 0.11 release, while we update more components to the latest versions.

**Comment by aesedepece** [comment]
I'm resynchronizing a "fat wallet" now. Will likely take a couple of hours, bit I'll let you know how it turns out!

**Comment by aesedepece** [comment]
> I'm resynchronizing a "fat wallet" now. Will likely take a couple of hours, bit I'll let you know how it turns out!

It catched the genesis transaction but then ignored any further transaction in this wallet.

**Comment by aesedepece** [comment]
> > I'm resynchronizing a "fat wallet" now. Will likely take a couple of hours, bit I'll let you know how it turns out!
> 
> It catched the genesis transaction but then ignored any further transaction in this wallet.

The issue was:

```
The transaction ... already exists in the database
```

This is an old friend of us, right?

**Comment by tmpolaczyk** [comment]
> > > I'm resynchronizing a "fat wallet" now. Will likely take a couple of hours, bit I'll let you know how it turns out!
> > 
> > 
> > It catched the genesis transaction but then ignored any further transaction in this wallet.
> 
> The issue was:
> 
> ```
> The transaction ... already exists in the database
> ```
> 
> This is an old friend of us, right? @mariocao

Opened #1818 to discuss this

**Pull request merged**